src/scores/languages.py

```diff
@@ -0,0 +1,17 @@
+"""Scoring language info."""
+
+from pandas import DataFrame
+
+
+def main(df: DataFrame):
```

Comment:
My preference is to avoid being too verbose with type annotations. If Python can automatically infer the output type, I think it's cleaner to let it do that. In the Ruff linting rules, I've only turned on type annotations for arguments (`ANN0`). If we want to do that for return types as well, we can switch to all the rules (`ANN`) to make sure we're consistent everywhere.

<img width="538" alt="Screenshot 2024-09-01 at 12 05 10" src="https://github.com/user-attachments/assets/151486a6-0346-4945-94b1-f9d01813041a">

Reply:
This is a common debate, and I don't have a strong opinion on it. On the one hand, IDEs / Ruff / mypy / etc can often correctly infer return type. On the other hand, sometimes they get it wrong, adding it can improve readability, and can also make debugging faster as the type checker shouts about changes in return type. Also, the zen of Python says explicit is better than implicit. For these reasons, I usually prefer an explicit return type, and I find it helpful in the example you highlight in my MR where I've gotten the type wrong. But if you've got strong opinions on this, then I'm happy to remove return types too

Reply:
Ok, let's go explicit then!